README.md

```diff
@@ -84,18 +84,18 @@ this works.
 ## YAML Configuration files
```

Comment:
I think this section could have been replaced with a link to https://kernelci.org/docs/core/config/

In fact, maybe this README should be entirely rewritten in a much smaller version now that we have more documentation available.